tours/cytof/app.R

```diff
@@ -3,6 +3,9 @@ library(iSEE)
 tour_steps <- read.delim("tour.txt", header = TRUE, as.is = TRUE, sep = ";")
 bcrxl <- readRDS("sce.rds")
 
+# Reset the path to the HDF5 file relative to the container
+path(assay(bcrxl, "exprs")) <- "/app/sce.hdf5"
```

Comment:
That's not going to work on the Shiny server.

Reply:
You could just do `file.path(getwd(), "sce.hdf5")`. And HDF5 file extensions are usually `*.h5`.

Reply:
Thanks for the catch @LTLA 
See commit below.
For some reason, the containerised cytof tour still crashes with error:
```
Error in .BLOCK_unique(x, incomparables = incomparables, ...) : 
  unused argument (nmax = nmax)
```
While the TCGA doesn't!
As far as I can see they're both preprocessed identically. I genuinely can't spot where the issue is.
It has to be a Docker container issue, because both tours work fine from an interactive R session.